File: MangaFox/app/src/main/java/com/example/doanthanhthai/mangafox/model/Season.java
package com.example.doanthanhthai.mangafox.model;

import java.io.Serializable;
import java.util.List;

/**
 * Created by ThaiDT1 on 7/17/2018.
 */

public class Season implements Serializable {
    public String name;
    public String url;
    public List<Episode> episodeList;
}

File: MangaFox/app/src/main/java/com/example/doanthanhthai/mangafox/model/Season.kt
package com.example.doanthanhthai.mangafox.model

import java.io.Serializable

/**
 * Created by ThaiDT1 on 7/17/2018.
 */

class Season : Serializable {
    var name: String? = null
    var url: String? = null
    var episodeList: List<Episode>? = null
}
